src/police.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-import roslib; roslib.load_manifest('rgb_lights_manager')
```

Comment:
Was this just a line that wasn't used for anything?

Reply:
Indeed.